lib/screenplay/suppressed_predictions.ex

```diff
@@ -62,7 +62,7 @@ defmodule Screenplay.SuppressedPredictions do
     line_stops_map =
       PredictionSuppression.line_stops()
       # End stations don't suppress predictions, filter them out
-      |> Enum.filter(&(&1.type != :end))
+      |> Enum.filter(&(&1.suppression_type != nil))
```

Comment:
The comment here talks about "end stations" but that made more sense when `:end` was the value being filtered on; could use an update or maybe just removing it, since I think the new condition speaks for itself.